eventsDB.js

```diff
@@ -119,7 +119,7 @@ updateRouter.post('/updateEvent', (req) => {
 });
 
 // Handling DELETE request.
-deleteRouter.delete('/deleteEvent', (req) => {
+deleteRouter.delete('/deleteEvent', (req, res) => {
```

Comment:
Thanks for fixing the response codes by adding res to the parameters

Reply:
also, agree nice fix

Reply:
Thank-you guys